requirements.txt

```diff
@@ -8,7 +8,8 @@ Jinja2>=2.7.3
 Sphinx>=1.2.3
 docopt>=0.6.2
 simplejson>=3.8.1
+requests>=2.9.1
 
 # Python3 compat
 future>=0.15.2
-configparser>=3.3.0; python_version < '3'
+#configparser>=3.3.0; python_version < '3'
```

Comment:
I thought about that. Mageia as embeded configparser into future.
But this is not the case for fedora and mainly pypi. (see below)

I'm not opposed to put PBMACRO into requirements.txt, but it means that you will have to generate the pypi stuff. So right now I would not go in this direction.
Last time you put a sed patch within the spec file if i'm not wrong. For 0.3 I will go for this.
More other this problem will not remain anymore as soons as distro will embed a recent setuptools.



[uggla@ugglalaptop ~]$ pip show configparser
You are using pip version 7.1.0, however version 8.1.1 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
---
Metadata-Version: 1.1
Name: configparser
Version: 3.5.0b2
Summary: This library brings the updated configparser from Python 3.5 to Python 2.6-3.5.
Home-page: http://docs.python.org/3/library/configparser.html
Author: Lukasz Langa
Author-email: lukasz@langa.pl
License: MIT
Location: /usr/lib/python2.7/site-packages
Requires: